README.md

```diff
@@ -16,12 +16,16 @@
 - HTML
 - CSS
 - Git
+- JavaSCript
+
```

Comment:
- [ ] OPTIONAL
It's a nice and smart thing to give appropriate credit to **Cindy Shin**, the author of the original design, as required by the Creative Commons License. Kindly ensure to add this under the "Acknowledgements" section of your ReadMe

Reply:
Done, thx 😋